src/panels/lovelace/editor/config-elements/hui-area-card-editor.ts

```diff
@@ -154,6 +174,21 @@ export class HuiAreaCardEditor
                       },
                     ] as const satisfies readonly HaFormSchema[])
                   : []),
+                ...(displayType === "camera"
+                  ? ([
+                      {
+                        name: "camera_entity",
+                        required: true,
+                        default: cameraEntities?.[0],
+                        selector: {
+                          entity: {
+                            include_entities: cameraEntities,
+                            filter: [{ domain: "camera" }],
```

Comment:
I think you don't need a domain filter if you've already whitelisted the list of cameras?